gems/quilt_rails/lib/generators/quilt_rails/install_generator.rb

```diff
@@ -2,9 +2,16 @@
 
 module QuiltRails
   class InstallGenerator < Rails::Generators::Base
-    def run_all_generators
-      generate("sewing_kit:install")
-      generate("quilt:install")
+    def setup_sewing_kit_gem
+      say("Add sewing_kit gem")
+      gem("sewing_kit")
+    end
+
+    def run_react_app_generator
+      after_bundle do
```

Comment:
You don't get this for free. It might be better to just declare a dependency on `sewing_kit`.

Reply:
`quilt_rails` are meant to be decouple if someone want to use it that way (external developer who have no access to sewing-kit) so I dont want to have quilt_rails be dependent on sewing-kit if someone does not need it.

I can remove this task and having the consumer run the two generator manually from the documentation maybe?

Reply:
If that's the case then the sewing_kit generator should be optional, yes. This generator doesn't serve much purpose at this point I think. Maybe we should remove it?

Reply:
I mainly kept it around so it wont be a breaking change. It certainly looks like it is necessary now to do a breaking change and remove it.